app/views/external_pledges/create.js.erb

```diff
@@ -1 +1,2 @@
-// rerender #existing-external-pledges
\ No newline at end of file
+$('#existing-external-pledges').empty().append('<%= escape_javascript(render partial: "external_pledges/external_pledges_table", locals: { patient: @patient, external_pledges: @patient.external_pledges.order_by('created_at desc') } ) %>');
+$('#new-external-pledge').empty().append('<%= escape_javascript(render partial: "external_pledges/new", locals: { patient: @patient, new_external_pledge: @new_external_pledge } ) %>');
```

Comment:
Probably what's going on here is that this instance variable is now a persisted object, so you'll want to flush this out with a new object. I recommend popping out `new_external_pledge: @new_external_pledge` with just simply `new_external_pledge: @patient.external_pledges.new`. So you'll get a line as follows:

```
$('#new-external-pledge').empty().append('<%= escape_javascript(render partial: "external_pledges/new", locals: { patient: @patient, new_external_pledge: @patient.external_pledges.new } ) %>');
```